apps/frontend/src/app/components/user-search/user-search.component.html

```diff
@@ -0,0 +1,25 @@
+<form [formGroup]="userSearchForm">
+  <nb-form-field>
+    <nb-icon *ngIf="isSteamId" class="steam-icon" nbPrefix nbPopover="Searches on SteamID" nbPopoverTrigger="hint"
```

Comment:
`steam-icon` class is unused. Also let's do "Search by steamID64".

Also, just realising we're not showing how to use this feature until you actually add the `steam:` prefix. Maybe we should add a small little Steam icon that displays a tooltip like "Prefix with steam: to search by steamID64", could have pressing the icon insert the prefix. Maybe having that left-aligned would look a bit weird, I'd expect tooltip icons on the right of a text input. Idk, could discuss more in Discord.